indico/modules/events/registration/templates/display/participant_list.html

```diff
@@ -11,6 +11,14 @@
     {% endif %}
 {% endblock %}
 
+{% block description %}
+    {%- if tables|length > 1 -%}
+        {% trans -%}
+            The lists of participants grouped by the registration form they used to registered to the event.
```

Comment:
But is it really relevant that they are grouped by registration forms and not by "whatever the managers decided"?